### PR TITLE
add 3 point assem. and release notes to SDK

### PR DESCRIPTION
**Pull request by jerrycowell96**

### CODE REVIEW

File: website/docs/hardware-tools/three-point.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+point lift.
+
+:::caution
+If you are missing any necessary pieces, please contact [farm-ng](https://amiga.farm-ng.com/docs/support/)
+immediately.
+:::
+
+## Included Hardware
+
+1. **24** Carriage bolts 3/8" by 1 1/14"
```
Comment (Hackerman342) [inline website/docs/hardware-tools/three-point.md:17]: Should this be 1 1/4" ??? 
```suggestion
1. **24** Carriage bolts 3/8" by 1 1/4"
```

File: website/docs/hardware-tools/three-point.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+10. Attach a CAN cable to the CAN hub.
+     <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/3e652b3a-9456-4ae1-a032-57fab9f0b86b"
+     width="320"
+     height="200" />
+11. Connect E-stop and CAN cables to H-bridge bringing power to it from the main harness.
+     <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/f511a27b-5d08-427e-bcf4-e42fc5444ba8"
+     width="500"
+     height="200" />
+
+12. Connect power to the 3point cylinder ( please note lower connection).
```
Comment (Hackerman342) [inline website/docs/hardware-tools/three-point.md:112]: Does that top connector just stay exposed? Seems dangerous / very not water proof

File: website/docs/hardware-tools/three-point.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+     <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/25db5975-db23-472d-8359-db72afbdb3eb"
+     width="280"
+     height="200" />
+
+8. Attach the adjustable top link to the 3 point backplate.
+     <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/277f2a9d-bdd1-4648-8061-231c078561fa"
+     width="320"
+     height="200" />
+
+9. Add T to the existing E-stop connection.
```
Comment (Hackerman342) [inline website/docs/hardware-tools/three-point.md:98]: Where do we find this T ?

**Review by Hackerman342 — APPROVED** [review]
Somethings are unclear to me, but if @TravisJoe or @AgustinBRodriguez think it should be clear enough for the customer then feel free to merge

File: website/docs/hardware-tools/three-point.md (hunk 2 above)
Comment (jerrycowell96) [reply website/docs/hardware-tools/three-point.md:120]: that is the intended design right now. maybe will change in the future but as far as the instructions go it is accurate

**Comment by Hackerman342** [comment]
@jerrycowell96 do you think this should be under the `Amiga Rover` sidebar section rather than `Development Tools`?

**Comment by jerrycowell96** [comment]
No. The Amiga rover section is for the base Amiga. Because not everyone gets the three point it would best fit under, development tools with this like microcontroller kit and the custom cables.

**Comment by AgustinBRodriguez** [comment]
There is a 3 point lift kit under farm tools; perhaps a link to the
installation guide on there would be helpful

**Comment by Hackerman342** [comment]
Yes great idea @AgustinBRodriguez. There should be a link on the farm-ng.com website to the page. Can you do that @jerrycowell96 or does someone else edit that website?

This really is not a developer tool, so @jerrycowell96 I suggest to add a new section for `Farm Tools`that right now will only have 3 point, but maybe soon will also have info about PTO / compost spreader. Make it a closer match to the sections on the main website.

**Comment by jerrycowell96** [comment]
Sure think. I can add that to the website and the section to the SDK

**Pull request merged**